app/controllers/users_controller.rb

```diff
@@ -24,11 +38,17 @@ def show
     @following_relationship = Relationship.new
   end
 
-  def update
-  end
-
   private
   def user_params
     params.require(:user).permit(:full_name, :password, :email)
   end
+
+  def handle_invitation
+    if params[:invitation_token].present?
+      invitation = Invitation.where(token: params[:invitation_token]).first
```

Comment:
You should use `find_by` instead of `where().first`. This was mentioned in previous PR and review.